calico_kubernetes/logutils.py

```diff
@@ -3,7 +3,7 @@
 import os
 import sys
 
-LOG_DIR = '/var/log/calico/kubernetes'
+LOG_DIR = '/var/log/calico/kubernetes/exec-plugin/'
```

Comment:
This was based on a comment from Casey in the original Issue. I think he had a future process like the Policy Agent in mind.